tests/parquet_test.py

```diff
@@ -423,27 +423,29 @@ def test_segarray_write(self):
             rd_data = ak.read_parquet(f"{tmp_dirname}/int_test*")
             for i in range(3):
                 self.assertListEqual(s[i].to_list(), rd_data[i].to_list())
+        print("Finished Int")
```

Comment:
```suggestion

```

Reply:
Manually removed this when resolving merge conflicts.